pages/profile/roles/remove/content/index.js

```diff
@@ -2,13 +2,13 @@ const { merge } = require('lodash');
 const baseContent = require('../../../content');
 
 module.exports = merge({}, baseContent, {
-  title: 'Please select the role you want to remove',
+  title: 'Amend profile',
```

Comment:
That's going to leave this page very sparse in terms of actual context. It's just going to be "Amend profile" and a list of roles with radio buttons, and no explanation of what a user is actually _doing_. Is that intentional?